rcl_yaml_param_parser/test/test_parse_yaml.cpp

```diff
@@ -508,8 +507,59 @@ TEST(test_file_parser, maximum_number_parameters) {
   {
     rcl_yaml_node_struct_fini(params_hdl);
   });
-  bool res = rcl_parse_yaml_file(path, params_hdl);
-  EXPECT_FALSE(res);
+  EXPECT_TRUE(rcl_parse_yaml_file(path, params_hdl));
+}
+
+TEST(test_file_parser, multiple_number_nodes) {
+  rcutils_reset_error();
+  EXPECT_TRUE(rcutils_get_cwd(cur_dir, 1024)) << rcutils_get_error_string().str;
+  rcutils_allocator_t allocator = rcutils_get_default_allocator();
+  char * test_path = rcutils_join_path(cur_dir, "test", allocator);
+  ASSERT_TRUE(NULL != test_path) << rcutils_get_error_string().str;
+  OSRF_TESTING_TOOLS_CPP_SCOPE_EXIT(
+  {
+    allocator.deallocate(test_path, allocator.state);
+  });
+  char * path = rcutils_join_path(test_path, "multiple_nodes.yaml", allocator);
```

Comment:
Same question here: do you mean to test this both here and in the separate file.  If so, why?